.env

```diff
@@ -19,8 +19,11 @@ _APP_INFLUXDB_HOST=influxdb
 _APP_INFLUXDB_PORT=8086
 _APP_STATSD_HOST=telegraf
 _APP_STATSD_PORT=8125
-_APP_SMTP_HOST=maildev
-_APP_SMTP_PORT=25
+_APP_SMTP_HOST=smtp.sendgrid.net
```

Comment:
Key the default `maildev` hostname and port 25. Try to pass empty values for the other vars, I'm not sure if ti will work but lets see if the tests pass.

Reply:
maybe we can declare var and keep empty after =, so it will be empty